application/controllers/EventRuleController.php

```diff
@@ -283,7 +281,7 @@ public static function createFilterString(Filter\Rule $filters): string
 
         $filterStr = QueryString::render($filters);
 
-        return ! empty($filterStr) ? $filterStr : '';
+        return ! empty($filterStr) ? rawurldecode($filterStr) : '';
```

Comment:
This is to store and display certain special characters like '/' as it is instead of in its url encoded form. This was earlier done in `setObjectString` method in `EventRuleObjectFilter`.